src/app/screens/createInscription/ContentLabel/index.tsx

```diff
@@ -0,0 +1,236 @@
+import axios from 'axios';
+import { MouseEvent as ReactMouseEvent, useEffect, useMemo, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+import DotsVerticalIcon from '@assets/img/dots_three_vertical.svg';
+import EyeIcon from '@assets/img/eye.svg';
+import ShareIcon from '@assets/img/share.svg';
```

Comment:
FYI we are in the middle of migrating to phosphor icon set, see here for usage:
https://github.com/secretkeylabs/xverse-web-extension/blob/069859a25e37c2dbbaabc93a151be1dcc45189fc/src/app/components/seedPhraseInput/index.tsx#L69

Reply:
🙌  That's awesome. Will update it now.